web/help/ztest.md

```diff
@@ -0,0 +1 @@
+test file
```

Comment:
Did you mean to delete this test file?

Reply:
I did mean to. It was just to make sure comited and pushed okay.